docs/source/conf.py

```diff
@@ -181,6 +181,9 @@ def setup(sphinx):
     sphinx.connect("autoapi-skip-member", skip_log)
 
 
+# Used to make version number available as a global variable in pages
+rst_epilog = f".. |version| replace:: {version}"
+
```

Comment:
Do we need this? `|version|` and `|release|` are default substitutions already provided: https://www.sphinx-doc.org/en/master/usage/restructuredtext/roles.html#substitutions

Reply:
Thanks for catching that. Will test without this.